server/services/gameLogic.js

```diff
@@ -170,6 +170,21 @@ exports.voteCard = (username, cardId) => {
     }
 }
 
+/*  */
+exports.emitVotes = () => {
+    setStatus(statusTypes.DISPLAY_ALL_VOTES);
+    socket.emitStatus(status);
+    socket.emitAllVotes(votes);
+    calcScores();
+    setTimeout(() => nextRound(), 5000);
+};
+
+exports.emitPlayedCards = () => {
+    status = statusTypes.WAITING_FOR_VOTES;
```

Comment:
Are these two functions setting `status` in different ways? I have no conext for this so could be completely wrong.

Reply:
What do you mean?

Reply:
No it's the same, I think things got changed around while we were debugging but now we're just doing `status = ` everywhere?